package.json

```diff
@@ -1,6 +1,10 @@
 {
   "private": true,
-  "repository": "github.com:3scale/system.git",
+  "repository": {
```

Comment:
Not a problem to follow this format, but we could do it shorter `"repository": "github.com:3scale/porta.git",` as seen in the [documentation](https://docs.npmjs.com/files/package.json).

![image](https://user-images.githubusercontent.com/11318903/50628040-9725d380-0f36-11e9-9482-696fc9445e5f.png)

Reply:
+1

Reply:
The reason was to make the https explicit, because of this comment here https://github.com/3scale/porta/pull/459#discussion_r244705485